### PR TITLE
[10.0] add new module to manage seasonal products

### PR DESCRIPTION
**Pull request by Cedric-Pigeon**
Coming from https://github.com/OCA/product-attribute/pull/117 and adapted to V10

### CODE REVIEW

File: product_life_period/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+* Define a life period
+* Assign a life period to a product
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to ...
```
Comment (LoisRForgeFlow) [inline product_life_period/README.rst]: Could you fill this?

File: product_life_period/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/135/8.0
```
Comment (LoisRForgeFlow) [inline product_life_period/README.rst]: 10.0

**Comment by rousseldenis** [comment]
@Cedric-Pigeon Is this still applicable ?

**Comment by Cedric-Pigeon** [comment]
@rousseldenis Yes we still use it

File: product_life_period/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2015  Laetitia Gangloff, Acsone SA/NV (http://www.acsone.eu)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Product life period",
+    "version": "10.0.1.0.0",
+    'author': "Acsone SA/NV, Odoo Community Association (OCA)",
+    "category": "Product",
+    "website": "http://www.acsone.eu",
```
Comment (rousseldenis) [inline product_life_period/__manifest__.py]: Please change this key

File: product_life_period/models/product_life_period.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+    @api.multi
+    def _run_life_period_update(self):
+        """ At the end of life period, set product in state end
+            And set the period inactive
+        """
+        end_life_periods = self.search(
+            [('end_date', '<=', fields.Date.today())])
+        for end_life_period in end_life_periods:
+            end_life_period.product_ids.write({'state': 'end'})
```
Comment (rousseldenis) [inline product_life_period/models/product_life_period.py]: Build product ids set and write on all once.

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Some comments

**Comment by rousseldenis** [comment]
@LoisRForgeFlow Could you update your review ?

**Review by rousseldenis — APPROVED** [review]

**Review by lmignon — APPROVED** [review]
Thank you @Cedric-Pigeon  LGTM (Code review only)

**Pull request merged**